app/Http/Controllers/MessagesController.php

```diff
@@ -81,11 +81,11 @@ public function show($id){
         return view('messenger.show', compact('thread', 'users'));
     }
 
-    public function update($id){
+    public function update($id) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->